backport/backport.ts

```diff
@@ -374,6 +374,12 @@ const backport = async ({
 
 	if (!merged) {
 		console.log('PR not merged')
+		await github.issues.createComment({
```

Comment:
I'm not 100% sure but this might result in a new comment every time a new label is set on the PR. Especially early one in the life of the PR lots of workflows potentially manipulate the labels so this might become an issue. Is there perhaps an easy way to determine if the comment is already there?

Reply:
@zerok valid concern. Ideally I'd want to post it more than just once, say once per day. It could be easy to lose the previous instance of the comment on a long lived PR, and then I might once again forget that the PR has to be merged before the backport PR will be opened :joy: 

However, my latest commit should do for now. I'm trying to post the comment _only if_  it hasn't been posted before. Unfortunately I don't see a nice way to test this in the code, any ideas there?

Reply:
I usually just try these things in a private clone of grafana/grafana (or even simpler repos) but IMO your change should work. It might have some false-positives but in this case that shouldn't matter 👍